src/sass/partials/_sidebar.scss

```diff
@@ -21,4 +21,13 @@ aside.pf-c-page__sidebar {
             .ins-c-navigation__header-title { @include rem('width', 160px); }
         }
     }
+
+    .ins-c-page__documentation a:before {
```

Comment:
Can we add some vertical padding on this?

Reply:
JustDoIt ... NIKE

Reply:
It follows PF guidelines. Going to approve.